docs/howto/configure/simple_extensions.md

```diff
@@ -13,7 +13,7 @@ also work with JupyterLite.
 
 ### Creating a new environment
 
-The easiest way to add new extensions is to use the `jupyterlite` CLI in a Python
+The easiest way to add new extensions is to use the `jupyterlite-core` CLI in a Python
```

Comment:
are we actually renaming the CLI? that will break a lot of things.

Reply:
no it was just to avoid confusion about the package name.

We could say "the JupyterLite CLI" instead to remove ambiguity.

Reply:
Maybe

> _the `jupyter lite` CLI_

?